flight_phase_files/HALO_RF02_20200122_info.yaml

```
@@ -24,59 +24,125 @@ remarks:
 - Descending Cirrus (9-13.5km) coming from West and South of the circle.
 - Away from the southwest edge of the track the air was almost entirely cloud free.
 segments:
-- kind: circle
-  name: c1
-  start: 2020-01-22 15:18:17
-  end: 2020-01-22 16:13:02
-- kind: circle_break
-  name: cb1to2
-  start: 2020-01-22 16:13:02
-  end: 2020-01-22 16:31:19
-- kind: circle
-  name: c2
-  start: 2020-01-22 16:31:19
-  end: 2020-01-22 17:26:29
-- kind: circle_break
-  name: cb2to3
-  start: 2020-01-22 17:26:29
-  end: 2020-01-22 17:43:08
-- kind: circle
-  name: c3
-  start: 2020-01-22 17:43:08
-  end: 2020-01-22 18:38:02
-- kind: baccardi_calibration
-  name: baccardi_cal
+- kinds:
+  - circle
+  name: Circle 1
+  irregularities:
+  - Circle is  363.52 deg to accomodate all 12 dropsondes
+  segment_id: HALO-0122_c1
+  start: 2020-01-22 15:17:17
+  end: 2020-01-22 16:13:05
+  good_dropsondes: 12
+- kinds:
+  - circle_break
+  name: Break between circle 1 and 2
+  irregularities: []
+  segment_id: HALO-0122_cb1
+  start: 2020-01-22 16:13:05
+  end: 2020-01-22 16:30:19
+- kinds:
+  - circle
+  name: Circle 2
+  irregularities: []
+  segment_id: HALO-0122_c2
+  start: 2020-01-22 16:30:19
+  end: 2020-01-22 17:26:44
+  good_dropsondes: 12
+- kinds:
+  - circle_break
+  name: Break between circle 2 and 3
+  irregularities: []
+  segment_id: HALO-0122_cb2
+  start: 2020-01-22 17:26:44
```

Circle 2 can be closed:
```suggestion
  end: 2020-01-22 17:26:51
  good_dropsondes: 12
- kinds:
  - circle_break
  name: Break between circle 2 and 3
  irregularities: []
  segment_id: HALO-0122_cb2
  start: 2020-01-22 17:26:51
```

---

yes, good idea. I had to shift around the times a bit to get rid of this annoying ;-) yellow warning message that the circle didn't start one minute before